demos/create_workspace.ipynb

```diff
@@ -0,0 +1,164 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### This notebook sets up workspaces for a new user. Can only be run by someone with sufficient access."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from adam import ConfigManager\n",
+    "from adam import Service\n",
+    "from adam import Permission"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Edit this to reflect the user and parent workspace for which you would like to add a workspace.\n",
+    "\n",
+    "Note: it's unusual for dev and prod to have the same uuids for anything - they happen to have the same uuids for a few objects created early on in the life of ADAM because at one point we copied the table contents. For newly created objects uuids will be assigned independently."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "name = # Fill this in!\n",
```

Comment:
Add a comment or example: is this name the person name (First Last)? the project name? github id? can it have spaces?